packages/web-components/src/components/va-promo-banner/va-promo-banner.tsx

```diff
@@ -0,0 +1,170 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  h,
+  Prop,
+  State,
+} from '@stencil/core';
+
+const DISMISSED_PROMO_BANNERS_LOCAL_STORAGE_KEY = 'DISMISSED_PROMO_BANNERS';
+@Component({
+  tag: 'va-promo-banner',
+  styleUrl: 'va-promo-banner.css',
+  shadow: true,
+})
+export class VaPromoBanner {
+  @Element() el: HTMLElement;
+
+  /**
+   * The rendering custom markup via the slot instead of the `<a>` with `text` in it
+   * */
+  @Prop() renderCustom: boolean = false;
+
+  /**
+   * `href` attribute for the `<a>` tag. Only gets used if `render` is _not_ used
```

Comment:
Should this be `renderCustom`?

Reply:
So it was originally `render` but I thought that was a confusing prop name since theres a render function, but this is supposed to allow for a prop that indicates custom html that isn't inside of an anchor tag. Def open to suggestions on renaming it

Reply:
Oh, I meant renaming `render` to `renderCustom` in the comment to reflect the latest prop name.

Reply:
Updates have been made